query/src/test/scala/filodb/query/exec/StitchRvsExecSpec.scala

```diff
@@ -141,7 +141,7 @@ class StitchRvsExecSpec extends AnyFunSpec with Matchers with ScalaFutures {
     mergeAndValidate(rvs, expected)
   }
 
-  it ("should reduce output range") {
+  it ("should output range passed in to StitchRvsExec") {
```

Comment:
This is the only positive test case I see. Any other edge cases that we need to validate ?